packages/default-theme/logic/checkout/useUICheckoutPage.js

```diff
@@ -80,12 +81,15 @@ export const useUICheckoutPage = () => {
       currentStep.value === CHECKOUT_STEPS.REVIEW &&
       nextStepNumber > CHECKOUT_STEPS.REVIEW
     ) {
-      await createOrder()
+      const order = await createOrder()
+      vm.$router.push(`/success-page?orderId=${order.id}`)
```

Comment:
you may import PAGE_SUCCESS_PAGE from `~helpers/pages.js`

and then
```suggestion
      vm.$router.push(`/${PAGE_SUCCESS_PAGE}?orderId=${order.id}`)
```

Reply:
done https://github.com/DivanteLtd/shopware-pwa/pull/709/commits/947dece8f1f55e1112d8431dfb82b67906b58e9f